sdk/compute/Azure.ResourceManager.Compute/api/Azure.ResourceManager.Compute.netstandard2.0.cs

```diff
@@ -3425,7 +3467,13 @@ public partial class GalleryDiskImage
         public GalleryDiskImage() { }
         public Azure.ResourceManager.Compute.Models.HostCaching? HostCaching { get { throw null; } set { } }
         public int? SizeInGB { get { throw null; } }
-        public Azure.ResourceManager.Compute.Models.GalleryArtifactVersionSource Source { get { throw null; } set { } }
+        public Azure.ResourceManager.Compute.Models.GalleryDiskImageSource Source { get { throw null; } set { } }
```

Comment:
This is a breaking change and we need to resolve it in the coming update of compute